Comments/components/commentBrowser.js

```diff
@@ -65,49 +78,63 @@ export class CommentBrowser extends Window {
       clipMode: 'auto',
       name: 'comment container'
     });
-    this.containerLayout = new Morph({
+    this.layoutContainer = new Morph({
       layout: new VerticalLayout({
         spacing: 5,
         orderByIndex: true
       }),
       name: 'comment container layout'
     });
-    this.container.addMorph(this.containerLayout);
+    this.container.addMorph(this.layoutContainer);
     this.addMorph(this.container);
   }
 
   close () {
-    super.close();
+    // super.close();
     const topbar = $world.getSubmorphNamed('lively top bar');
     if (topbar) {
       topbar.uncolorCommentBrowserButton();
     }
-    this.removeCommentIndicators();
-    instance = undefined;
+    this.remove();
   }
 
-  async refreshCommentGroupMorphs () {
+  async initializeCommentGroupMorphs () {
     const commentGroupMorphs = [];
     await Promise.all($world.withAllSubmorphsDo(async (morph) => {
       if (morph.comments.length == 0) {
         return;
       }
+      morph.comments.forEach(async (comment) => {
+        await this.addCommentForMorph(comment, morph);
+      });
+    }));
+  }
+
+  async addCommentForMorph (comment, morph) {
+    if (morph.id in this.commentGroups) {
+      await this.commentGroups[morph.id].addCommentMorph(comment);
+      this.commentGroups[morph.id].relayout();
+    } else {
       const commentGroupMorph = await resource('part://CommentGroupMorphMockup/comment group morph master').read();
```

Comment:
Maybe add a TODO Comment here that we want to adjust the location of the master? Robin had no time to explain this to us today, but I would argue that we want to have this inside of a lively.colab/comments/components folder in the long run? Same for other places where master-components are loaded.

Reply:
ok

Reply:
We added comments in the commentMorph.js and commentBrowser.js